src/integrationTest/java/uk/gov/hmcts/reform/pip/rules/controllers/ControllerTest.java

```diff
@@ -51,13 +51,43 @@ public void publicationThatExistsTest() throws Exception {
         final int publicationID = 1;
         assertEquals(publicationID, publication.getPublicationId(), "Publication ID is " + publicationID);
         assertEquals(1, publication.getCourtHearingsList().size(), "The array contains a single publication");
+    }
+
+    @DisplayName("Should return a 200, with a publication that contains a single court list")
+    @Test
+    public void publicationThatExistsHearingsListTest() throws Exception {
+        MvcResult response = mockMvc.perform(get("/publication/1")).andExpect(status().isOk()).andReturn();
+
+        assertEquals(200, response.getResponse().getStatus(),
+                     "Check that a 200 status is returned is returned");
+
+        ObjectMapper objectMapper = new ObjectMapper();
+        objectMapper.registerModule(new JavaTimeModule());
+        Publication publication = objectMapper.readValue(
+            response.getResponse().getContentAsByteArray(), Publication.class);
 
         List<CourtHearings> courtHearingsList = publication.getCourtHearingsList();
         assertThat(courtHearingsList.get(0).getCourtId()).as("Get first court id").isEqualTo(1);
+    }
+
+    @DisplayName("Should return a 200, with a publication that contains a multiple hearings")
+    @Test
+    public void publicationThatExistsHearingsTest() throws Exception {
+        MvcResult response = mockMvc.perform(get("/publication/1")).andExpect(status().isOk()).andReturn();
+
+        assertEquals(200, response.getResponse().getStatus(),
```

Comment:
is this testing the same as line 76? in both youre only checkint the status, maybe return earlier on line 76 and do the assertion here or vice versa

Reply:
Good spot :) I've removed the checks via the mockMvc response, and done it via the assert instead.

I've updated the other tests in this file to do that too.